lib/resources/tasks/run.js

```diff
@@ -1,22 +1,10 @@
 import gulp from 'gulp';
 import browserSync from 'browser-sync';
 import historyApiFallback from 'connect-history-api-fallback/lib';
+import { CLIOptions } from 'aurelia-cli';
```

Comment:
So it's an Aurelia standard? Hm, I wonder if the Aurelia VS Code plugin would be willing to accommodate this so that VS Code's autoformat feature will still work...